<p><b>[WARN]</b>&nbsp;Retrying (Retry(total=2, connect=None, read=None, redirect=None, status=None)) after connection broken by 'NewConnectionError('<urllib3.connection.HTTPConnection object at 0x10f9344c0>: Failed to establish a new connection: [Errno 61] Connection refused')': /session/2dd383807708324c50f1e1226346e8c3</p><p><b>[WARN]</b>&nbsp;Retrying (Retry(total=1, connect=None, read=None, redirect=None, status=None)) after connection broken by 'NewConnectionError('<urllib3.connection.HTTPConnection object at 0x10f9fc940>: Failed to establish a new connection: [Errno 61] Connection refused')': /session/2dd383807708324c50f1e1226346e8c3</p><p><b>[WARN]</b>&nbsp;Retrying (Retry(total=0, connect=None, read=None, redirect=None, status=None)) after connection broken by 'NewConnectionError('<urllib3.connection.HTTPConnection object at 0x10f9fca90>: Failed to establish a new connection: [Errno 61] Connection refused')': /session/2dd383807708324c50f1e1226346e8c3</p>